nym-vpn-core/crates/nym-gateway-directory/src/gateway_client.rs

```diff
@@ -107,7 +116,7 @@ impl Config {
 
         // Since harbourmatser isn't part of the standard nym network details, we need to handle it
         // as a special case.
-        let harbour_master_url = if network.network_name == "mainnet" {
+        let harbour_master_url = if network.network_name == defaults::mainnet::NETWORK_NAME {
```

Comment:
if we can do the same for harbourmaster that would be great


        let harbour_master_url = if network.network_name == defaults::mainnet::NETWORK_NAME {
            Some(
                nym_vpn_api_client::MAINNET_NYM_HARBOURMASTER_URL
                    .parse()
                    .expect("mainnet default nym-harbourmaster-url not parseable"),
            )
        } else {
            std::env::var("MAINNET_NYM_HARBOURMASTER_URL").ok().map(|url| {
                url.parse()
                    .expect("MAINNET_NYM_HARBOURMASTER_URL env variable not a valid URL")
            })
        };

Reply:
All of this will be deleted in the follow up PR as remove any reference to harbourmaster from the vpn client